README.md

```diff
@@ -2,7 +2,6 @@
 [![@articulate/sox](https://img.shields.io/npm/v/@articulate/sox.svg)](https://www.npmjs.com/package/@articulate/sox)
 [![Build Status](https://travis-ci.org/articulate/sox.svg?branch=master)](https://travis-ci.org/articulate/sox)
 [![Coverage Status](https://coveralls.io/repos/github/articulate/sox/badge.svg?branch=master)](https://coveralls.io/github/articulate/sox?branch=master)
-[![NSP Status](https://nodesecurity.io/orgs/articulate/projects/4cf8d77e-4275-42d3-b2a4-8c2e1fa790d4/badge)](https://nodesecurity.io/orgs/articulate/projects/4cf8d77e-4275-42d3-b2a4-8c2e1fa790d4)
```

Comment:
Why are we removing this ? Is it addressed with this update?

Reply:
nodesecurity.io does not exist anymore